### ISSUE TITLE
Street names missing in places like Ireland

### ISSUE THREAD

**Issue by evilexecutable** [openmaptiles/openmaptiles#1074]
- Clear description of the problem: What object is missing/extra/wrong? What did you expect to be in the map instead?
We have generated maps for Ireland, and certain street names are not showing. For example, you can see this happen on the public OMT site:
https://www.maptiler.com/maps/#basic//vector/16.56/-6.277889/53.344502

I read in another issue that this can be changed in an SQL file, however since recent changes this file no longer exists and so I'm not sure what configuration I need to change in order to get this fixed.

![omt_missing_names](https://user-images.githubusercontent.com/6544968/106763415-b5d38880-662e-11eb-88bc-176aa156ecc6.png)

**Comment by TomPohys** [comment]
Hi @evilexecutable, there are a lot of `service` (or `residential`) roads without any name in your example. Recently there was a change for stitching together the roads https://github.com/openmaptiles/openmaptiles/commit/45f5b53c9c6ad754b052b09bd4c76ca01f8eed8f, which increased the quality of stitching the roads by name. But if you would like to add more names you can play with zoom levels in https://github.com/openmaptiles/openmaptiles/blob/master/layers/transportation_name/transportation_name.sql or eventually with the style itself.

I hope it will help.

**Comment by evilexecutable** [comment]
> 
> 
> Hi @evilexecutable, there are a lot of `service` (or `residential`) roads without any name in your example. Recently there was a change for stitching together the roads [45f5b53](https://github.com/openmaptiles/openmaptiles/commit/45f5b53c9c6ad754b052b09bd4c76ca01f8eed8f), which increased the quality of stitching the roads by name. But if you would like to add more names you can play with zoom levels in https://github.com/openmaptiles/openmaptiles/blob/master/layers/transportation_name/transportation_name.sql or eventually with the style itself.
> 
> I hope it will help.

Thanks @TomPohys, it gives me somewhere to look. If I'm able to do this in the style, I assume the roads I'm looking at would fall under "highway-name-minor" or "highway-name-path"?

**Comment by TomPohys** [comment]
Hi @evilexecutable, If I look at OSM data, for `service` or `residential` type of roads there is no name for them. So, could you please provide some more detail? (e.g. which specific road name is missing).

![image](https://user-images.githubusercontent.com/5182210/110094001-9d749c00-7d9b-11eb-953c-494496eccbdf.png)

Thanks

**Comment by TomPohys** [comment]
Hi @evilexecutable, could you please provide more information? e.g. which road name on which road is missing. Thanks.

**Comment by evilexecutable** [comment]
Hi @TomPohys thanks for getting back.

Unfortunately I know longer work for the company that I raised this issue on behalf of, however I do remember it was most rural streets in Northern Ireland. We did also find that once we added other styles to the front end, the street names were OK, so I noted it down as a style related problem.

To be perfectly honest I forgot I had even raised this issue, I think it can probably be closed as it doesn't look like a problem with OMT.
